static/css/components/work--tablet.less

```
@@ -4,7 +4,6 @@
 // These elements are all children.
 div.edition {
   &Cover {
-    margin-bottom: 1.5em;
     padding-right: 0;
   }
   &About {
```

```suggestion
  &About {
    .book-description {
      clear: both;
    }
```
Adding the clear here will prevent the layout issue in tablet views.  Please add this below the `margin-top` rule on the next line.